pkg/model/nft_collection.go

```diff
@@ -47,6 +47,9 @@ type NFTCollectionDetail struct {
 }
 
 type NFTCollectionConfig struct {
-	NFTCollection
-	TokenID string `json:"token_id"`
+	ID        uuid.NullUUID `json:"id"`
```

Comment:
should add swaggertype to convert uuid -> string on swagger

Reply:
this struct is only used for cronjob, i think we dont need add to swagger